studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -397,6 +397,11 @@ export const fillTimeseries = (
   })
 
   const diff = maxDate.diff(minDate, truncation as dayjs.UnitType)
+  if (diff > 10000) {
```

Comment:
ahh okay gotcha - let's leave a comment indicating this then

```suggestion
  // Intentional throwing of error here to be caught by Sentry, as this would indicate a bug since charts shouldn't be rendering more than 10k data points
  if (diff > 10000) {
```